src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Parser/ElementsResult.cs

```diff
@@ -12,11 +13,39 @@ public record ElementsResult : ParserStateResult
     public ElementsResult(ParserStateResult result)
         : base(result.FieldName, result.Result, result.ExplicitField, result.YieldReturn)
     {
+        this.Elements = new List<Element>();
```

Comment:
As mentioned elsewhere, this probably shouldn't be specific to SPDX 3.0

Reply:
Handled https://github.com/microsoft/sbom-tool/pull/993